app/Jobs/ApplicationDeploymentJob.php

```
@@ -1316,6 +1316,44 @@ private function laravel_finetunes()
+        if ($hasComposerInstall) {
+            $variables = data_get($parsed, 'variables', []);
+
+            $envCommands = [];
+            foreach ($variables as $key => $value) {
+                $envCommands[] = "echo \"$key=\${$key}\" >> /app/.env";
+            }
+
+            if (! empty($envCommands)) {
+                $checkSymfonyCmd = 'if [ -f /app/composer.json ] && (grep -q "symfony/dotenv\\|symfony/framework-bundle\\|symfony/flex" /app/composer.json 2>/dev/null); then touch /app/.env; fi';
+
+                $conditionalEnvCommands = [];
```

PHPMD (2.15.0)

Avoid excessively long variable names like $conditionalEnvCommands. Keep variable name length under 20. (undefined)

(LongVariable)